hi-ml-histopathology/src/histopathology/utils/girder.py

```diff
@@ -0,0 +1,466 @@
+#  ------------------------------------------------------------------------------------------
```

Comment:
Sone docstrings in this file could help in understanding the code better

Reply:
I've added some docs in 4625bc6.